tox.ini

```diff
@@ -23,12 +10,14 @@ deps =
     tutorials: jupyter
 extras =
     lint: dev
-    pytest: test
+    unit: test
+    integration: test
     minimum: test
 commands =
     lint: invoke lint
     readme: invoke readme
-    pytest: invoke pytest
+    unit: invoke unit
+    integration: invoke integration
     minimum: invoke minimum
     tutorials: invoke tutorials
```

Comment:
I do not think we have `tutorials` anymore, so maybe this and the other references to `tutorials` should be removed.